src/Symfony/Component/Intl/Countries.php

```diff
@@ -174,4 +245,11 @@ protected static function getPath(): string
     {
         return Intl::getDataDirectory().'/'.Intl::REGION_DIR;
     }
+
+    private static function ensureOptionalUserAssignedResolved(): void
+    {
+        if (!isset(self::$optionalUserAssigned)) {
+            self::$optionalUserAssigned = filter_var(getenv('SYMFONY_ALLOW_OPTIONAL_USER_ASSIGNED'), FILTER_VALIDATE_BOOLEAN);
+        }
+    }
```

Comment:
This method should return the value directly.

```suggestion
    private static function optionalUserAssigned(): bool
    {
        return self::$optionalUserAssigned ??= filter_var(getenv('SYMFONY_ALLOW_OPTIONAL_USER_ASSIGNED'), FILTER_VALIDATE_BOOLEAN);
    }
```

Reply:
This is great! Going to push an update tomorrow. Thanks <3